namespaces/live.cloud-platform.service.justice.gov.uk/laa-ccms-staging-file-upload-api-production/resources/serviceaccount.tf

```diff
@@ -0,0 +1,19 @@
+module "serviceaccount" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-serviceaccount?ref=1.1.0"
+
+  namespace          = var.namespace
+  kubernetes_cluster = var.kubernetes_cluster
+
+  github_actions_secret_kube_cert      = var.github_actions_secret_kube_cert
+  github_actions_secret_kube_token     = var.github_actions_secret_kube_token
+  github_actions_secret_kube_cluster   = var.github_actions_secret_kube_cluster
+  github_actions_secret_kube_namespace = var.github_actions_secret_kube_namespace
+  serviceaccount_token_rotated_date    = "01-01-2000"
+
+  # Uncomment and provide repository names to create github actions secrets
+  # containing the ca.crt and token for use in github actions CI/CD pipelines
+  # github_repositories = ["my-repo"]
+  github_repositories = ["laa-ccms-staging-file-upload-api"]
+  github_environments = ["production"]
```

Comment:
same here, checking production env in github

Reply:
yes we have a prod and production env